### PR TITLE
Removing veracode from reports.ts

### PR DESCRIPTION
**Pull request by Sandhya1874**

### CODE REVIEW

File: server/routes/veracode.ts [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import type { Services } from '../services'
+import { getFormattedName, veracodeFilters } from '../utils/utils'
+
+export default function routes({ serviceCatalogueService }: Services): Router {
+  const router = Router()
+
+  const get = (path: string, handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+  // const post = (path: string, handler: RequestHandler) => router.post(path, asyncMiddleware(handler))
+
+  get('/veracode', async (req, res) => {
```
Comment (psoleckimoj) [inline server/routes/veracode.ts:13]: So, you don't need `/veracode` here and all other routes in here, the initial `/veracode` is in `server/app.ts`. So basically `app.use('/veracode', veracodeRoutes(services))` is saying prefix every route in `veracodeRoutes` with `/veracode` which means what you've added here is `/veracode/veracode` 🙂 

Basically in this file remove `/veracode` in all the `get` and `post` statements you want to use 👍

File: server/routes/veracode.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+          report: hasVeracode ? component.attributes.veracode_results_url : 'N/A',
+          date: hasVeracode
+            ? dayjs(component.attributes.veracode_last_completed_scan_date).format('YYYY-MM-DD HH:mm')
+            : 'N/A',
+          codeScore: hasVeracode ? component.attributes.veracode_results_summary['static-analysis'].score : 0,
+          severityLevels,
+        }
+      })
+
+    return rows
```
Comment (psoleckimoj) [inline server/routes/veracode.ts:64]: Needs to be the same as the code you copied it from so `res.send(rows)` it's not a return 🙂

Comment (Sandhya1874) [reply server/routes/veracode.ts:64]: Thanks Paul , corrected it now

File: server/routes/veracode.ts (hunk 1 above)
Comment (Sandhya1874) [reply server/routes/veracode.ts]: Corrected

**Pull request merged**